build-system/tasks/presubmit-checks.js

```diff
@@ -317,6 +317,7 @@ var forbiddenTerms = {
       '3p/ampcontext.js',
       '3p/ampcontext-integration.js',
       'dist.3p/current/integration.js', // includes previous
+      'extensions/amp-analytics/0.1/transport.js',
```

Comment:
This exception is added because the amp-analytics tag needs to send messages to the third-party iframe. It covers the actual calls to the sendMessage() method.